src/mongo_logger.ts

```diff
@@ -797,12 +810,16 @@ export class MongoLogger {
     this.pendingLog = null;
   }
 
+  willLog(component: MongoLoggableComponent, severity: SeverityLevel): boolean {
+    return this.severities[component][severity];
```

Comment:
```suggestion
    if (severity === SeverityLevel.OFF) return false;
    return this.severities[component][severity];
```

Reply:
I see where you're coming from here. I've made the change.